project2/styles.css

```diff
@@ -0,0 +1,188 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    color: #686868;
+    font-size: 18px;
+}
+
+section {
+    background-color: #faf4e3;
+    max-width: 640px;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+header {
+    background-image: url(images/shrimps_2.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 300px;
+    border-radius: 5px;
+
+}
+
+.title {
+    display: block;
+    padding-left: 30px;
+    padding-top: 20px;
+}
+
+
+div {
+    height: 258px;
+    padding-left: 10px;
+}
+
+h2 {
+    text-align: left;
+    font-size: 42px;
+    color: #223b12;
+    font-family: "Notos Sans JP", sans-serif;
+    font-weight: 700;
+    margin-bottom: 10px;
+}
+
+.subhead {
+    text-align: left;
+    padding-left: 30px;
+}
+
+h3 {
+    text-align: right;
+    font-size: 22px;
+    font-family: "Notos Sans JP", sans-serif;
+    font-weight: 700;
+    padding-right: 30px;
+}
+
+h3:hover {
+    color: #223b12;
+    transition: 400ms;
+    text-decoration: underline;
+}
+
+h4 {
+    text-align: left;
+    font-size: 22px;
+    font-family: "Notos Sans JP", sans-serif;
+    font-weight: 700;
+    margin-top: 0px;
+
+}
+
+p {
+    text-align: left;
+    margin-right: 5%;
+    font-family: "Merriweather", serif;
+    font-weight: 400;
+}
+
+ul {
+    text-align: left;
+    column-count: 2;
+    column-gap: 35px;
+    font-family: "Merriweather", serif;
+    font-weight: 700;
+    line-height: 2.5;
+    font-size: 13px;
+    padding-left: 0px;
+}
+
+.button-reset {
```

Comment:
Two notes here:
1. There doesn't appear to be a button in your design, so this class can be deleted.
2. Even if the button did exist, you. want to rename this class -- "reset" named classes are meant to be multipurpose and about removing/undoing default styles, not applying new, specific styles. I can explain this more in class, if it helps.